website_and_docs/content/documentation/webdriver/elements/information.zh-cn.md

```diff
@@ -343,10 +384,9 @@ val text = driver.findElement(By.cssSelector("h1")).getText()
 {{< /tab >}}
 {{< /tabpane >}}
 
-## 获取特性或属性
+## 获取属性
 
-获取与 DOM 属性关联的运行时的值.
-它返回与元素的 DOM 特性或属性关联的数据. 
+获取DOM属性相关的运行时值。它返回与该元素的DOM属性或属性相关的数据。
```

Comment:
This is a new function in Selenium 4, it's better to diff the difference in translation.  I can't out the value of overwriting totally...

Reply:
```suggestion
获取与 DOM 属性关联的运行时的值。
它返回与该元素的 DOM 特性或属性关联的数据。
```

I'm not sure what you mean, maybe I should unify the wording?

Reply:
it's fine now, can you commit the change?